### PR TITLE
feat: Deploy script

### LINKED ISSUE #610: metrics deploy command
**Author the user story for this feature**

As a user, I want to automatically deploy metrics so I can use the app.

**Describe the solution you'd like**

Simple version of deploying the metrics app. To be improved in future tickets.

`metrics deploy`

### PR DESCRIPTION
**Pull request by radekdymacz**
Fixes #610

## Checklist

- [ ] Documented
- [ ] Unit tested
- [ ] Component tested
- [ ] Driver tested

### CODE REVIEW

**Comment by bootstraponline** [comment]
I think the CLI should be called `metrics`, then we can have `metrics deploy`, `metrics doctor`, etc.

File: deploy/test/deploy_test.dart [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import 'package:deploy/deploy.dart';
+import 'package:test/test.dart';
+
+void main() {
+  test('calculate', () {
```
Comment (bootstraponline) [inline deploy/test/deploy_test.dart]: should we delete this? I think it's auto generated from when the project was created.

File: deploy/Makefile [hunk 2]
```diff
@@ -0,0 +1,6 @@
+.PHONY: build
+
+build:
+	 rm -rf build
+	 mkdir build
+	 dart2native bin/deploy.dart -o build/metrics
```
Comment (bootstraponline) [inline deploy/Makefile]: image

let's add a newline

**Comment by radekdymacz** [comment]
closed in favour of #610

**Pull request closed without merging**